AIFG-3D-Connect-4/FourTechRulesHandler.cpp

```diff
@@ -4,10 +4,10 @@
 FourTechRulesHandler::FourTechRulesHandler(GameBoard & t_board) :
 	m_TOTAL_BOARD_TILES{ 64u },
 	m_board{ t_board },
-	m_piecesPlaced{ 0u },
-	m_ai{ t_board },
-	m_playersTurn{ true }
+	m_piecesPlaced{ 0u }
 {
+	m_turnHandler.setPlayer1(new ConsoleInput(t_board));
```

Comment:
I'd do this setup in Game.cpp before passing the input handler to the rules handler.